DAOS-17203 build: Build DAOS RPMs with Leap 15.6

phender opened the pull request:
Use the opensuse-leap-15.6-x86_64.cfg mock config to build DAOS RPMs.

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test: true
Skip-func-test-leap15: false
Skip-func-test-el8: true

Requires:

- [ ] https://github.com/daos-stack/pipeline-lib/pull/460

### Steps for the author:

* [ ] Commit message follows the [guidelines](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Appropriate [Features or Test-tag](https://daosio.atlassian.net/wiki/spaces/DC/pages/10984259629/Test+Tags) pragmas were used.
* [ ] Appropriate [Functional Test Stages](https://daosio.atlassian.net/wiki/spaces/DC/pages/12147556353/CI+Functional+Test+Stages) were run.
* [ ] At least two positive code reviews including at least one code owner from each category referenced in the PR.
* [ ] Testing is complete. If necessary, forced-landing label added and a reason added in a comment.

#### After all prior steps are complete:
* [ ] Gatekeeper requested (daos-gatekeeper added as a reviewer).

phender committed:
DAOS-17203 build: Build DAOS RPMs with Leap 15.6

Use the opensuse-leap-15.6-x86_64.cfg mock config to build DAOS RPMs.

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test: true
Skip-func-test-leap15: false
Skip-func-test-el8: true

Signed-off-by: Phil Henderson <phillip.henderson@hpe.com>

daosbuild3 commented:
Test stage Build RPM on Leap 15.6 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/1/execution/node/328/log

daosbuild3 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/1/execution/node/360/log

daosbuild3 commented:
Test stage Build on Leap 15.6 with Intel-C and TARGET_PREFIX completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/1/execution/node/556/log

daosbuild3 commented:
Test stage Build RPM on EL 9 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/2/execution/node/346/log

daosbuild3 commented:
Test stage Build on Leap 15.6 with Intel-C and TARGET_PREFIX completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/2/execution/node/594/log

phender committed:
Merge branch 'master' into hendersp/DAOS-17302_2

phender committed:
Ensure we have lua-lmod > 8.7.34.

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test: true
Skip-func-test-leap15: false
Skip-func-test-el8: true

Signed-off-by: Phil Henderson <phillip.henderson@hpe.com>

daosbuild3 commented:
Test stage Build RPM on Leap 15.6 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/3/execution/node/360/log

daosbuild3 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/3/execution/node/317/log

phender committed:
Attempt 2.

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test: true
Skip-func-test-leap15: false
Skip-func-test-el8: true

Signed-off-by: Phil Henderson <phillip.henderson@hpe.com>

daosbuild3 commented:
Test stage Build RPM on Leap 15.6 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/4/execution/node/357/log

daosbuild3 commented:
Test stage Build RPM on EL 9 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/4/execution/node/351/log

daosbuild3 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/4/execution/node/360/log

daosbuild3 commented:
Test stage Build on Leap 15.6 with Intel-C and TARGET_PREFIX completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-16648/4/execution/node/608/log
